includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -262,6 +262,26 @@ public function maybe_process_pre_orders( $order_id ) {
 		);
 	}
 
+	/**
+	 * Customer param wrong? The user may have been deleted on stripe's end. Remove customer_id. Can be retried without.
+	 *
+	 * @since 4.2.0
+	 * @param object   $error The error that was returned from Stripe's API.
+	 * @param WC_Order $order The order those payment is being processed.
+	 * @return bool           A flag that indicates that the customer does not exist and should be removed.
+	 */
+	public function maybe_remove_non_existent_customer( $error, $order ) {
```

Comment:
Not something we should change but why was this function ever public. protected seems most appropriate but probably should leave public

Reply:
I moved this from `WC_Gateway_Stripe` to `WC_Stripe_Payment_Gateway`, so that both `WC_Gateway_Stripe` and `WC_Stripe_UPE_Payment_Gateway` could inherit and lay claim to it. I didn't really think much of it, but totally agree that `protected` seems more appropriate to me. However, that being said, in the name of backwards compatibility, perhaps it should remain `public` just in case any external code is already calling this function for some odd reason. 🤷